CredentialProvider/View/CredentialItemListView.swift

```diff
@@ -23,4 +23,35 @@ class ItemListView: BaseItemListView, ItemListViewProtocol {
     override func createPresenter() -> BaseItemListPresenter {
         return ItemListPresenter(view: self)
     }
+
+    override func styleNavigationBar() {
+        super.styleNavigationBar()
+
+        self.navigationItem.leftBarButtonItem = UIBarButtonItem(customView: self.cancelButton)
+
+        if let presenter = presenter {
+            (self.navigationItem.leftBarButtonItem?.customView as? UIButton)?.rx.tap
```

Comment:
Rather than all this `?`, I usually just make the button like: 
```
guard let presenter = self.presenter else { return }

let cancelButton = self.cancelButton
self.navigationItem.leftBarButtonItem = cancelButton
cancelButton.rx.tap......
```